### PR TITLE
Dedicated OA System Serving Multiple Air Handlers

### PR DESCRIPTION
**Pull request by rongpengzhang**
Pull request overview
---------------------
This new feature will allow EnergyPlus to model an outside air system supplying ventilation air to multiple air handlers. Currently a DOAS in EnergyPlus is a normal AirLoopHVAC (aka air loop)  object with the recirculated air flow set to zero. In EnergyPlus the air loops are independent of each other: one air loop can not provide air to another. Outside air is provided to each air system by the AirLoopHVAC:OutdoorAirSystem object. Each outdoor air system can supply only one air loop. All this means that EnergyPlus can't simulate a common HVAC layout: a single multi-floor DOAS serving, through an air shaft, multiple single floor air handling units.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Code style (parentheses padding, variable names)
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check -- verify this
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add rules to spreadsheet
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Documentation changes in place
 - [ ] Changed docs build successfully
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

**Comment by nrel-bot-3** [comment]
@wfbuhl @rongpengzhang @lgentile it has been 14 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@wfbuhl @rongpengzhang @lgentile it has been 14 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@wfbuhl @rongpengzhang @lgentile it has been 14 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@wfbuhl @rongpengzhang @lgentile it has been 14 days since this pull request was last updated.

File: src/EnergyPlus/MixedAir.cc [hunk 1]
```diff
@@ -206,6 +206,9 @@ namespace MixedAir {
 	int const ControllerOutsideAir( 2 );
 	int const ControllerStandAloneERV( 3 );
 
+	int const RegularOASys( 1 );
+	int const DOASToMultAirSys( 2 );
```
Comment (rraustad) [inline src/EnergyPlus/MixedAir.cc:210]: Other than GetInput, where are these new types used?

**Comment by rraustad** [comment]
@rongpengzhang can you clarify these changes. I see this description in the design doc, but can't find where this happens in the code. Can you point me to where in code this happens?

    +We will use the existing EnergyPlus object AirLoopHVAC:OutdoorAirSystem to describe the DOAS serving multiple air handlers. Let's call this use of AirLoopHVAC:OutdoorAirSystem an Outdoor Air System to Multiple Air Handlers (OAStoMAH) vs Outdoor Air System in Single Air Loop (OASinSAL). Just as  AirLoopHVAC:OutdoorAirSystem can currently supply treated outdoor air to a single AirLoopHVAC the new use of  AirLoopHVAC:OutdoorAirSystem will supply treated outdoor air to multiple AirLoopHVAC's. This would correspond in the real world to a DOAS supplying multiple air handlers. In the EnergyPlus model each AirLoopHVAC will still need an AirLoopHVAC:OutdoorAirSystem. But additionally an AirLoopHVAC:OutdoorAirSystem can connect to each AirLoopHVAC through the outdoor air inlet of the OutdoorAir:Mixer contained in its AirLoopHVAC:OutdoorAirSystem.
     +
     +**Since the OAStoMAH will supply multiple AirLoopHVAC's, we need a splitter to divide its output among the AirLoopHVAC's. Similarly we will need to use a mixer to combine the relief air coming from the AirLoopHVAC's and send it to the OAStoMAH. We will borrow from AirLoopHVAC:ZoneSplitter and AirLoopHVAC:ZoneMixer but will define new objects AirLoopHVAC:OutdoorAirSystem:Mixer and AirLoopHVAC:OutdoorAirSystem:Splitter. As noted above, we will add an additional outside air node to Outdoor:AirMixer to allow for economizer action.**
     +

**Comment by nrel-bot-3** [comment]
@wfbuhl @rongpengzhang @lgentile it has been 14 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@wfbuhl @rongpengzhang @lgentile it has been 14 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@wfbuhl @rongpengzhang @lgentile it has been 14 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@wfbuhl @rongpengzhang @lgentile it has been 14 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@wfbuhl @rongpengzhang @lgentile it has been 14 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@wfbuhl @rongpengzhang @lgentile it has been 14 days since this pull request was last updated.

**Comment by nrel-bot-3** [comment]
@wfbuhl @rongpengzhang @lgentile it has been 14 days since this pull request was last updated.

**Comment by Myoldmopar** [comment]
Closing this as it has sat without any review or effort for a long time now (almost a year!!).  I'll leave the branch on the repo so that someone can pick this up anytime and reopen a pull request.

**Pull request closed without merging**